pom.xml

```diff
@@ -34,6 +34,7 @@
     <!-- TODO https://issues.redhat.com/browse/PLANNER-2042 -->
     <spotbugs.failOnViolation>false</spotbugs.failOnViolation>
     <version.org.mockito>3.1.0</version.org.mockito>
+    <optaplanner-spring-boot-school-timetabling-quickstart.version>0.1.0-SNAPSHOT</optaplanner-spring-boot-school-timetabling-quickstart.version>
```

Comment:
one additional comment to standards of property names. I would suggest to use always the same pattern
version.groupId[.more-identifying-id]

Reply:
done, rnamed to `version.com.example.optaplanner-school-timetabling-quickstart`